api/v1alpha1/component.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2023 The Kubebb Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+const (
+	ComponentFinalizer = "kubebb.component.finalizer"
```

Comment:
use `Finalizer`  https://github.com/kubebb/core/blob/main/api/v1alpha1/common.go#LL31C1-L31C1 instead

Reply:
see https://github.com/kubebb/core/pull/17#discussion_r1226142242 and https://github.com/kubebb/core/pull/17#discussion_r1226152694